wallet/src/controller.rs

```diff
@@ -627,6 +627,10 @@ where
 			ok(create_error_response(e))
 		}))
 	}
+
```

Comment:
So with the OPTIONS HTTP method is supposed to return the list of allowed HTTP methods on that endpoint in a `Allow` header. Perhaps add that?

Reply:
Would it be ok if I opened a separate PR for adding the appropriate HTTP methods in the header? I ask because the `foreign_api` is currently unusable, and this patch brings it in to working condition and parity with the `owner_api` behavior (which also doesn't return the methods). 

Note: I'm in the process of learning Rust right now, so it's slow going to patch even the basics. But I'm also working to get a client wallet in to a good place for the protocol launch this week, so I was prioritizing getting the API up to a basic functioning state. I'll happily open a PR to address your comment in when I have a moment .

Thoughts?

Reply:
Seems reasonable, given how small a change this it.